dashboard/models.py

```diff
@@ -139,14 +139,19 @@ class Meta:
 
 
 class CourseQuerySet(models.QuerySet):
-    def get_supported_courses(self) -> QuerySet:
+    def get_supported_courses(self, include_last_updated=False) -> QuerySet:
         """Returns the list of supported courses from the database
 
+        :param include_last_updated: Whether or not to include the last updated value in the return, otherwise just returns the id's
         :return: [List of supported course ids]
         :rtype: [list of str (possibly incremented depending on parameter)]
         """
         try:
-            return self.values_list('id', flat=True)
+            # If we want to include the last updated date, otherwise just return a flat set
+            if include_last_updated:
+                return self.values_list('id', 'data_last_updated')
+            else:
+                return self.values_list('id', flat=True)
```

Comment:
This might be trying to do too many things. It would be nice if the queryset always returned data with one shape. Django has a pretty good API, so I'd be included to keep this as it was, and then just use `Course.objects.values_list(`id`, `data_last_updated`)`.

I don't remember, when does this `model.DoesNotExist` error occur?

Reply:
I agree, I've simplified this back to how it was and updated it in the cron. 

The blame of this DoesNotExist goes back to me in 2018. I don't remember if it was needed or not. 793128b0f5dd2003d595781752ff18a422be1806